src/fontra/views/editor/editor.js

```diff
@@ -452,6 +454,48 @@ export class EditorController {
     this.initSidebarTextEntry();
   }
 
+  initSidebarGutters() {
+    let initialWidth;
+    let initialPointerCoordinateX;
```

Comment:
There is a few ways to achieve the resize, I chose the easiest I could. @justvanrossum.

Calculating the new width without setting the initial state of the element and cursor in `onpointerdown` is possible, but it requires a calculation of the clicked point on the gutter relatively to the sidebar in onpointermove. Otherwise it flickers couple of pixels (gutter size flickering). Example how it must be done https://stackoverflow.com/a/52910578/17805504, here how it must not be done https://stackoverflow.com/a/34487057/17805504 see the flickering.